api/graphql/schema.graphql

```diff
@@ -306,6 +306,7 @@ enum PoolAdvertisementLanguage {
 type PoolCandidate {
     id: ID!
     pool: Pool @belongsTo
+    poolAdvertisement: PoolAdvertisement @belongsTo(relation: "pool")
```

Comment:
I'm not sure if this is correct but not sure how else to get the `essentialSkills` and `nonessentialSkills`

Reply:
I think this is correct. Even though pool and poolAdvertisement are technically fetching from the same model, this makes it more explicit. Also, I believe the Pool type doesn't have `essentialSkills` and `nonEssentialSkills`.

Reply:
Thanks, I was a little unsure about this and wanted to confirm I wasn't doing something incorrect.